blocklist-client/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "blocklist-client"
+version = "0.0.1"
+edition = "2021"
+
+[[bin]]
+name = "blocklist-client"
+path = "src/main.rs"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+warp = "0.3"
```

Comment:
Although `warp` seems okay, I think we should go with `axum`, mainly because that repo is seeing much more activity and is more used by others. I haven't looked into how warp is doing so I'm saying I see problems with using it, just that it's something we should be mindful of.

Reply:
I prefer `warp` over `axum`. I have worked with it in larger projects before and it has never let me down.